input_parser.star

```diff
@@ -411,7 +410,7 @@ DEFAULT_ARGS = (
         # The type of consensus contract to use.
         # Consensus Options:
         # - 'rollup': Transaction data is stored on-chain on L1.
-        # - 'cdk_validium': Transaction data is stored off-chain using the CDK DA layer and a DAC.
+        # - 'cdk-validium': Transaction data is stored off-chain using the CDK DA layer and a DAC.
```

Comment:
```suggestion
        # - 'cdk_validium': Transaction data is stored off-chain using the CDK DA layer and a DAC.
```

Reply:
this is gonna be addressed in another [PR](https://github.com/0xPolygon/kurtosis-cdk/pull/616)